examples/shapes_and_collections/plot_path_patch.py

```diff
@@ -2,6 +2,9 @@
 ================
 PathPatch object
 ================
+
+This example shows how to create `Path`\s and `PathPatch` objects through
+Matplotlib's API.
 """
```

Comment:
I am not sure I understand the question. If you are asking about the need for blanklines around `"""`, then the answer is no.